cloudstack/resource_cloudstack_network.go

```diff
@@ -176,14 +175,12 @@ func resourceCloudStackNetworkCreate(d *schema.ResourceData, meta interface{}) e
 		return e.Error()
 	}
 
-	// Compute/set the display text
-	displaytext, ok := d.GetOk("display_text")
-	if !ok {
-		displaytext = name
-	}
-
 	// Create a new parameter struct
-	p := cs.Network.NewCreateNetworkParams(displaytext.(string), name, networkofferingid, zoneid)
+	p := cs.Network.NewCreateNetworkParams(name, networkofferingid, zoneid)
+
+	if displaytext, ok := d.GetOk("display_text"); ok {
+		p.SetDisplaytext(displaytext.(string))
```

Comment:
@fabiomatavelli We are changing the existing behavior of terraform provider here.
Previously we used to set the value of `displaytext` same as `name` if `displaytext` wasn't set.
This could break existing user's setup.

Reply:
Fixed, I've added an `else` condition to set the display text with the `name` in case the `display_text` param is not set.